free-1.md

```diff
@@ -45,6 +56,8 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+I think this is a good possibility for a project because there just does not seem to be something like this out there. FLStudio and similar applications allow you to pitch bend and autotune, which is nice, but there isn't anything out there exactly like what I am describing. This makes me both hopeful and nervous about this idea.
```

Comment:
I've seen some small DSL languages that are useful for spoofing your voice in online games. That being said, I don't believe these are geared towards music as much voice disguising.